gcp-deployer/README.md

```diff
@@ -104,3 +104,27 @@ The yaml is unmarshalled into a [machinesetup.configList](../cloud/google/machin
 Right now, the startup scripts must be bash scripts. 
 There is a set of environment variables that is concatenated to the beginning of the startup script.
 You can find them [here](../cloud/google/metadata.go) and use them in your startup script as needed.
+
+## Contributing
+
+### Building
+To build a binary, run the following command from this folder.
+
+```
+go build .
+```
+
+### Testing
+When changing this application, you will often end up modifying other packages above this folder in the project tree. Before 
+opening any PR you should run all the unit tests in the repository. To run the unit tests, run the following command from the 
+root, ```cluster-api```, folder of the repo.
+
+```
+go test ./...
```

Comment:
Won't this run your integration test as well? It would be good if there is a way to only run intergration tests or only run unit tests.

Reply:
It will not run the integration tests. You must pass -tags=integration to run the integration tests. Unfortunately, I cannot find a way to ONLY run the integration tests without requiring an invariant / standard that all unit tests have their own build tag.

I did a bunch of research on this and it was the best I could come up with -- if you can find a better way I'm happy to change. Ideally you would be able to JUST run the integration tests but unfortunately that doesn't seem to be easy.